app.py

```diff
@@ -30,6 +32,12 @@
 app = Flask(__name__)
 app.config.from_prefixed_env()
 log = app.logger
+limiter = Limiter(
+    get_remote_address,
+    app=app,
+    default_limits=["200 per day", "50 per hour"],
+    storage_uri="redis://redis:6379/",
```

Comment:
Students are unlikely to have redis provisioned in their "production" environment. I understand this is backwards. Can you ommit L39?

Then we can control the behaviour using environment variables instead: 

local environment
`RATELIMIT_STORAGE_URI="redis://redis:6379/"`

production environment
`RATELIMIT_STORAGE_URI="memory://"`